app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,107 @@
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	xmlns:tools="http://schemas.android.com/tools"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:gravity="center"
+	android:orientation="vertical">
+
+	<ImageView
+		android:id="@+id/vLoginLogoCover"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:background="@drawable/login_cover"
+		android:src="@drawable/login_cover_gradient"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toTopOf="parent" />
+
+	<ImageView
+		android:id="@+id/vLoginWoloxLogo"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_alignParentStart="true"
+		android:layout_marginTop="@dimen/login_cover_logo_height"
+		android:adjustViewBounds="false"
+		android:gravity="bottom|center_horizontal"
+		android:src="@drawable/wolox_logo_large"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintHorizontal_bias="0.0"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toTopOf="@+id/vLoginLogoCover" />
+
+	<EditText
+		android:id="@+id/vLogInEmail"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_marginTop="@dimen/spacing_large"
+		android:layout_marginStart="@dimen/spacing_large"
+		android:layout_marginEnd="@dimen/spacing_large"
+		android:gravity="center"
+		android:hint="@string/login_email"
+		android:importantForAutofill="no"
+		android:inputType="textEmailAddress"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintHorizontal_bias="0.0"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toBottomOf="@+id/vLoginWoloxLogo" />
+
+	<EditText
+		android:id="@+id/vLogInPassword"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_marginTop="@dimen/spacing_large"
+		android:layout_marginStart="@dimen/spacing_large"
+		android:layout_marginEnd="@dimen/spacing_large"
+		android:gravity="center"
+		android:importantForAutofill="no"
+		android:inputType="textWebPassword"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toBottomOf="@+id/vLogInEmail" />
+
+	<Button
+		android:id="@+id/vLogInButton"
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content"
+		android:layout_marginStart="@dimen/spacing_large"
+		android:layout_marginEnd="@dimen/spacing_large"
+		android:layout_marginTop="@dimen/spacing_large"
+		android:background="@color/blue"
+		android:gravity="center"
+		android:text="@string/login_button_login"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toBottomOf="@+id/vLogInPassword"
+		tools:layout_editor_absoluteX="16dp" />
```

Comment:
why did you use this tools? is not a good practice to leave the tools in xml, but if it is not necessary, you can delete it